lib/device_api/android/adb.rb

```diff
@@ -205,14 +205,14 @@ def self.screencap( serial, args )
         execute(cmd)
       end
 
-      # Returns the wifi status of device
+      # Returns wifi status and name of wifi
       # @param serial serial number of device
       # @example
       #   DeviceAPI::ADB.wifi(serial)
       def self.wifi(serial)
         result = execute("adb -s #{serial} shell dumpsys wifi | grep mNetworkInfo")
         raise ADBCommandError.new(result.stderr) if result.exit != 0
-        return result.match("state:(.*?),") result.match("extra:(.*?),")) if result.exit == 0
+        return [result.stdout.match("state:(.*?),")[0], result.stdout.match("extra:(.*?),")[0]] if result.exit == 0
```

Comment:
Could you return the information with keys please? It makes anything using this function a bit more readable. For example:
```
def get_wifi_status
  ADB.wifi(serial)[0]
end
```
vs 
```
def get_wifi_status
  ADB.wifi(serial)[:state]
end
```